dist/tools/openocd/openocd.sh

```diff
@@ -224,7 +234,9 @@ do_debugserver() {
 do_reset() {
     test_config
     # start OpenOCD and invoke board reset
-    sh -c "${OPENOCD} -f '${OPENOCD_CONFIG}' \
+    sh -c "${OPENOCD} \
+            ${OPENOCD_ADAPTER_INIT} \
+            -f '${OPENOCD_CONFIG}' \
```

Comment:
Isn't possible to factorise these 3 repeated commands?

Reply:
hmm, I don't see the benefit of concatenating three variables into one when it's only in 3-4 places. It would make the script less readable though in my opinion.